### PR TITLE
Lists component

### PR DESCRIPTION
**Pull request by sahiba0915**
Hello team ,please review my code.

Live Preview: https://lists-component-cl.netlify.app/

Files to check:
1. list.html
2. list.css

### CODE REVIEW

**Review by akash-1318 — COMMENTED** [review]
Code is really clean and well structured with proper comments for better understanding so from my end no changes require.

File: component/lists/list.html [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+                    <li class="list-items">Jam</li>
+                </ol>
+            </code>
+          </pre>
+          </div>
+          <hr />
+
+           <!-- End- Normal list -->
+          
+
```
Comment (chanchal-panpaliya) [inline component/lists/list.html:131]: format your code and remove unwanted space.

Comment (sahiba0915) [reply component/lists/list.html:131]: ok noted

**Review by chanchal-panpaliya — COMMENTED** [review]
code looks good just remove unwanted space And you not added 
list

**Pull request merged**